asabiris/output/smtp/service.py

```diff
@@ -125,6 +126,15 @@ async def send(
 		except aiosmtplib.errors.SMTPConnectError as e:
 			L.error("Connection failed: {}".format(e), struct_data={"host": self.Host, "port": self.Port})
 			raise SMTPDeliverError("SMTP delivery failed")
+		except aiosmtplib.errors.SMTPResponseException as e:
+			error_message = str(e)
+			if "Message size exceeds fixed limit" in error_message:
+				L.error("Message size exceeds fixed limit")
+				raise MessageSizeError("Message size exceeds fixed limit")
+			else:
+				L.error("An SMTP error occurred: {}; check credentials".format(e), struct_data={"host": self.Host})
+				raise SMTPDeliverError("SMTP delivery failed")
+
```

Comment:
Add also handling of `SMTPAuthenticationError` for invalid credentials.

+ make the generic handler to print this:

`L.exception("Generic error: {}".format(e), struct_data={"host": self.Host})`

Reply:
@ateska I can't reproduce this error to test it.